blinky/rel/config.exs

```diff
@@ -1,24 +1,23 @@
+# This sets the default release built by `mix release`
+# This sets the default environment used by `mix release`
```

Comment:
If the formatter wants these comments to be consolidated here, then we should make it a block comment that describes each setting, because now it's less obvious which one refers to which.

Since this code is generated by Distillery, you could also open a PR there for bonus points. 😁 🚀